dashboard/lib/policies/child_account.rb

```diff
@@ -192,7 +192,11 @@ def self.can_link_new_personal_account?(user)
 
   # Returns true if the user has provided the minimum information we need to decide if their account is affected by our Child Account Policy.
   def self.has_required_information?(user)
-    [user.us_state, user.country_code, user.birthday].all?(&:present?)
+    if ['US', 'RD'].include?(user.country_code)
```

Comment:
We use this check in many places, so I think it’s worth finally moving it to a separate method:
```ruby
class Policies::User
  def self.in_usa?(user)
    %w[US RD].include?(user.country_code)
  end
end
```
```suggestion
    if Policies::User.in_usa?(user)
```

Reply:
I think it makes even more sense to move this to the User model:
```ruby
class User
  def in_usa?
    %w[US RD].include?(country_code)
  end
end
```

Reply:
There is a similar method in `users_helper.rb`:
```
  def usa?(country_code)
    %w[US RD].include?(country_code.to_s.upcase)
  end
``` 

Difference being it takes the country code rather than the user as a param. Maybe we could just have one method to do this check to avoid duplicate code?

Reply:
I've changed it to use the `usa?` function from `users_helper.rb`

Reply:
Rails Helpers are primarily designed for use in Views to keep the logic out of templates and make them cleaner. While they can be used in Controllers, this is generally discouraged, as helpers are better suited for view-related tasks. For more details, see [this guide](https://www.rubyguides.com/2020/01/rails-helpers/)